CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contribuindo
+
+Uma vez que o _Idvogados_ é um projeto totalmente **open source**, qualquer pessoa/instituição pode contribuir da forma que desejar. Para isso basta seguir o guia de contribuição do projeto e nosso [código de conduta](https://github.com/idvogados/codigo-de-conduta/blob/master/README.md).
+
+_Issues_ ou _pull requests_ que não estejam de acordo com este guia serão recusados.
+
+## Linguagem
+
+- _Commits_ devem ser feitos em inglês.
+- _Issues_, _pull requests_ e revisões devem ser feitas em português para facilitar a comunicação e chegar a um consenso rapidamente.
+
+## Desenvolvimento
+
+- Desenvolva apenas em seu _branch_.
+- Faça _rebase_ com o **dev** **antes** de criar _pull requests_.
+- Faça _rebase_ do **dev** em seu _branch_ diariamente.
```

Comment:
Não acho que seja necessário fazer rebase todos os dias. Acredito que a pessoa garantir que tem as atualizações mais recentes da *dev* antes de começar a trabalhar já é o bastante.

Reply:
O 'diariamente' seria o caso do cenário ideal pra que não haja conflitos. Muitas das observações tirei de alguns exemplos que encontrei.  Acha que deveria remover essa linha?

Reply:
Acho que sim. Ou talvez trocar por `Mantenha seu fork sempre atualizado`. 
https://help.github.com/pt/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

Reply:
@pictos sobre essa questão, atualmente é uma das restrições nas branchs. Se for o caso se não termos essa restrição me avisem da decisão para eu remover da proteção das branchs.

![image](https://user-images.githubusercontent.com/3301060/80547061-a16eab00-898d-11ea-9206-0d1ef7462bbc.png)

Reply:
@rodrigondec, isso podemos ver no futuro. Por enquanto acho que pode deixar as configurações assim.